src/overlays/actors/ovl_Bg_Mizu_Movebg/z_bg_mizu_movebg.c

```diff
@@ -37,18 +37,28 @@ ActorInit Bg_Mizu_Movebg_InitVars = {
     (ActorFunc)BgMizuMovebg_Draw,
 };
 
-static f32 D_8089EB40[] = { -115.200005f, -115.200005f, -115.200005f, 0.0f };
-
-static Gfx* D_8089EB50[] = {
-    gObjectMizuObjectsMovebgDL_000190, gObjectMizuObjectsMovebgDL_000680, gObjectMizuObjectsMovebgDL_000C20,
-    gObjectMizuObjectsMovebgDL_002E10, gObjectMizuObjectsMovebgDL_002E10, gObjectMizuObjectsMovebgDL_002E10,
-    gObjectMizuObjectsMovebgDL_002E10, gObjectMizuObjectsMovebgDL_0011F0,
+static f32 dragonStatue1OffsetPosY[] = { -115.200005f, -115.200005f, -115.200005f, 0.0f };
+
+static Gfx* dLists[] = {
+    gWaterTempleFloatPlatformWestDL,                 // MIZUMOVEBG_TYPE_FLOAT_PLATFORM_WEST
+    gWaterTempleFloatPlatformOutsideCentralPillarDL, // MIZUMOVEBG_TYPE_FLOAT_PLATFORM_OUTSIDE_CENTER_PILLAR
+    gWaterTempleFloatPlatformInsideCentralPillarDL,  // MIZUMOVEBG_TYPE_FLOAT_PLATFORM_INSIDE_CENTER_PILLAR
```

Comment:
not super convinced about these names but I have no suggestion at the moment

maybe "float platform" could be "floating platform", I think we can afford three more letters

Reply:
I hate finding names, floating sounds better

I'd like to hear more opinions before changing this again though

Reply:
changed to floating